values.yaml

```diff
@@ -152,6 +158,16 @@ server:
     #    hosts:
     #      - chart-example.local
 
+  # OpenShift only - create a route to expose the service
+  # The created route will be of type reencrypt
+  # https://docs.openshift.com/container-platform/4.4/networking/routes/secured-routes.html#nw-ingress-creating-a-reencrypt-route-with-a-custom-certificate_secured-routes
+  # Services are secured by OpenShift "service-ca" controller
+  # https://docs.openshift.com/container-platform/4.4/authentication/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate
+  route:
+    enabled: true
+    labels: {}
+    annotations: {}
+    host: chart-example.local
```

Comment:
We might make a better default host here.  Maybe use the chart name?

Reply:
I used the same naming example as for the ingress, but I can update it